### PR TITLE
Add xpec for funcake_reqd_fields

### PR DESCRIPTION
**Pull request by htomren**

### CODE REVIEW

File: tests/schematron/funcake_reqd_fields.xpec [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+          <edm:isShownAt>http://digital.library.temple.edu/cdm/ref/collection/p16002coll25/id/0</edm:isShownAt>
+          <edm:preview>http://digital.library.temple.edu/cdm/ref/collection/p16002coll25/id/0/tnail</edm:preview>
+          <edm:dataProvider>Temple University</edm:dataProvider>
+          <edm:provider>PA Digital</edm:provider>
+        </oai_dc:dc>
+      </x:context>
+      <x:expect-valid/>
+    </x:scenario>
+
+    <x:scenario label="not valid: empty identifier>
```
Comment (lfinnigan) [inline tests/schematron/funcake_reqd_fields.xpec]: this quote needs a close

File: tests/schematron/funcake_reqd_fields.xpec [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            Collections Research Center, scrc@temple.edu, 215-204-8257.</dcterms:rights>
+          <schema:fileFormat>image/jp2</schema:fileFormat>
+          <dcterms:identifier/>
+          <edm:isShownAt>http://digital.library.temple.edu/cdm/ref/collection/p16002coll25/id/0</edm:isShownAt>
+          <edm:preview>http://digital.library.temple.edu/cdm/ref/collection/p16002coll25/id/0/tnail</edm:preview>
+          <edm:dataProvider>Temple University</edm:dataProvider>
+          <edm:provider>PA Digital</edm:provider>
+        </oai_dc:dc>
+      </x:context>
+      <x:expect-assert id="Identifier1"/>
```
Comment (lfinnigan) [inline tests/schematron/funcake_reqd_fields.xpec]: let's add a second one to look for <dcterms:identifier></dcterms:identifier>

File: tests/schematron/funcake_reqd_fields.xpec (hunk 1 above)
Comment (htomren) [reply tests/schematron/funcake_reqd_fields.xpec]: thanks!

File: tests/schematron/funcake_reqd_fields.xpec (hunk 2 above)
Comment (htomren) [reply tests/schematron/funcake_reqd_fields.xpec]: will do

**Pull request merged**